README.md

```diff
@@ -1,66 +1,66 @@
 [![HHVM Status](https://img.shields.io/hhvm/yiisoft/yii2-dev.svg)](http://hhvm.h4cc.de/package/yiisoft/yii2-dev)
 [![Code Coverage](https://scrutinizer-ci.com/g/yiisoft/yii2/badges/coverage.png?s=31d80f1036099e9d6a3e4d7738f6b000b3c3d10e)](https://scrutinizer-ci.com/g/yiisoft/yii2/)
 [![Scrutinizer Quality Score](https://scrutinizer-ci.com/g/yiisoft/yii2/badges/quality-score.png?s=b1074a1ff6d0b214d54fa5ab7abbb90fc092471d)](https://scrutinizer-ci.com/g/yiisoft/yii2/)
 [![Code Climate](https://img.shields.io/codeclimate/github/yiisoft/yii2.svg)](https://codeclimate.com/github/yiisoft/yii2)
 
-DIRECTORY STRUCTURE
--------------------
 
-```
-build/               internally used build tools
-docs/                documentation
-framework/           core framework code
-tests/               tests of the core framework code
-```
-
-
-REQUIREMENTS
+INSTALLATION
 ------------
 
-The minimum requirement by Yii is that your Web server supports PHP 5.4.
+**Requirements**
 
+Yii2 requires `PHP 5.4`. 
 
-DOCUMENTATION
--------------
+**Composer**
 
-Yii 2.0 has a [Definitive Guide](http://www.yiiframework.com/doc-2.0/guide-index.html) and 
-a [Class Reference](http://www.yiiframework.com/doc-2.0/index.html) which cover every detail of Yii.
+With [Composer](https://getcomposer.org/) installed, you can install Yii by running the following commands under a Web-accessible folder:
 
-There is also a [PDF version](http://stuff.cebe.cc/yii2-guide.en.pdf) of the Definitive Guide
-and a [Definitive Guide Mirror](http://stuff.cebe.cc/yii2docs/) which is updated every 15 minutes.
+```
+composer global require "fxp/composer-asset-plugin:^1.2.0"
+composer create-project --prefer-dist yiisoft/yii2-app-basic basic
+```
 
+More information on installing Yii2 can be found in our [installation manual](http://www.yiiframework.com/doc-2.0/guide-start-installation.html).
 For 1.1 users, you may refer to [Upgrading from Yii 1.1](docs/guide/intro-upgrade-from-v1.md)
 to have a general idea of what has changed in 2.0.
 
+DOCUMENTATION
+-------------
+Yii 2.0 has a Definitive Guide and a Class Reference API which cover every detail of Yii.
 
-HOW TO PARTICIPATE
-------------------
+There is also a PDF version of the Definitive Guide and a Definitive Guide Mirror which is updated every 15 minutes.
+
+| Type  | Online | PDF |
+| ------------- | ------------- | ------------- |
+| API | http://www.yiiframework.com/doc-2.0/index.html [(mirror)](http://stuff.cebe.cc/yii2docs/)| |
+| Guide |  http://www.yiiframework.com/doc-2.0/guide-index.html | http://stuff.cebe.cc/yii2-guide.en.pdf |
 
-### Your participation to Yii 2 development is very welcome!
 
-You may participate in the following ways:
+PARTICIPATION
+------------------
+Your participation to Yii 2 development is very welcome! There are many area's in which you can help us
+improve our framework:
+
+| Type  | Resource |
```

Comment:
Should it be a table? It looks like a list to me. Header and type aren't really helpful.